plugins/woocommerce/package.json

```
@@ -209,7 +209,7 @@
 						"start": "env:test"
 					},
 					"events": [
-						"pull_request",
+						"temp_skip_pull_request",
```

Revert before merging

```suggestion
						"pull_request",
```